crates/nu-parser/src/parser.rs

```diff
@@ -294,7 +294,7 @@ pub fn parse_external_call(
             let (arg, err) = parse_dollar_expr(working_set, *span, expand_aliases_denylist);
             error = error.or(err);
             args.push(arg);
-        } else if contents.starts_with(b"(") {
+        } else if contents.starts_with(b"[") {
```

Comment:
I think this might work (so you can pass `nu -c [ 'a' '-b' '--c' ]`, for example), but intstead of `parse_full_cell_path()`, I'd use `parse_list_expression()`.

I'm not sure why this conditional was there in the first place since it's already checked above. Maybe @jntrnr knows?